src/index.ts

```diff
@@ -187,12 +187,9 @@ export function declutter(arg: any): any {
   }
   let ret = { ...arg }
   // Marqeta can send back several empty fields on all responses... clean it
-  if (isEmpty(ret.nextUrl)) {
+  if (isEmpty(ret.metadata)) {
     delete ret.nextUrl
   }
-  if (isEmpty(ret.previousUrl)) {
-    delete ret.previousUrl
-  }
   if (ret.status === 200) {
     delete ret.status
   }
```

Comment:
Is this even in the data returned from Marqeta? You mentioned that there was no success flag... nothing in the output - but this would be just that success indicator - Right?

If it's not in the output, then there's no need to have it here... 

Are you sure you want to use this function? For Peach it made sense... and if we need it here to clean out the cruft, then OK... but I just want you to be sure.

Reply:
`previousUrl` is not used, will remove it.